alert.html

```diff
@@ -0,0 +1,44 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../vui-colors/colors.html">
+
+<dom-module id="vui-alert">
+    <template>
+        <style>
+            :root {
+                --very-red: #ff7962;
+                --less-red: #ffeeec;
+                --border-width: 5px;
+                --border-radius: 3px;
+                --padding: 6px;
+            }
+            
+            .container {
+                background-color: var(--less-red);
+                border-width: 0;
+                border-left: var(--border-width) solid var(--very-red);
+                border-bottom-right-radius: var(--border-radius);
+                border-top-right-radius: var(--border-radius);
+                padding: var(--padding);
+            }
+            
+            .container [dir='rtl'] {
+		        border-left-width: 0;
+		        border-right: var(--border-width) solid var(--very-red);
```

Comment:
I think what you are after is: `:host-context([dir="rtl"]) .container`.  This will target elements with the `.container` class that have, or has an ancestor that has `[dir='rtl']`.  While `[dir='rtl']` is valid on any element, we just put it on the `body`.